hare/flows_test.go

```diff
@@ -61,7 +56,7 @@ func (his *HareWrapper) waitForTermination() {
 			break
 		}
 
-		time.Sleep(300 * time.Millisecond)
```

Comment:
Why this change?

Reply:
its just checking whether outputs are there. since all the params are in seconds, i want it do less busy work.

why 300 millisecond in the first place i'd ask :D

Reply:
I don't believe the amount of work being done here in the check is significant compared to the rest of the work being done in these tests, so I'd be in favour of checking more often because that's going to make the tests complete faster.